src/drivers/github.go

```diff
@@ -8,21 +8,46 @@ import (
 
 	"golang.org/x/oauth2"
 
-	"github.com/git-town/git-town/src/git"
 	"github.com/google/go-github/github"
 )
 
+type githubConfig interface {
```

Comment:
These are the config values that the GitHub driver needs. Defining such driver-specific interfaces is optional, we could also just go with a `git.Configuration` instance here. This exists to help @blaggacao write lighter-weight tests.

Reply:
Thanks! It will help a great deal with mocking those calls. I also find the code more readable with interfaces.